src/async_client/client.py

```diff
@@ -84,7 +84,6 @@ async def _do_request_with_retries(self, method, path, **kwargs):
         @self.retrying.wraps
         async def wrapped(method, path, **kwargs):
             response = await self._session.request(method, path, **kwargs)
-            await response.aread()
```

Comment:
I was surprised that it works so I added a new test case: https://github.com/Toloka/toloka-kit/pull/185
Seems like it is true that this line is unnecessary